webxdc.d.ts

```diff
@@ -1,7 +1,7 @@
-type SendingStatusUpdate<T> = {
+type SendingStatusUpdate<PayloadType> = {
```

Comment:
I'm not a fan of these long types, but if you prefer them we can use them.

Reply:
I wrote this so people will understand what it is for in the future. 
I'm also lazy so I mostly use one letter generic attributes too, but that was not enough seemingly: https://github.com/webxdc/webxdc-types/pull/6#discussion_r1563980579 ;)